actix-web-codegen/src/lib.rs

```diff
@@ -17,10 +17,12 @@
 //! - [options](attr.options.html)
 //! - [trace](attr.trace.html)
 //! - [patch](attr.patch.html)
+//! - [route](attr.route.html)
 //!
 //! ### Attributes:
```

Comment:
it's not clear this section should be here any more

Reply:
Shall I take them all down, or just remove `route`?

Reply:
yeah i think remove it and move the notes section